dev_guide/builds/triggering_builds.adoc

```diff
@@ -32,6 +32,12 @@ link:https://docs.gitlab.com/ce/user/project/integrations/webhooks.html[GitLab],
 link:https://confluence.atlassian.com/bitbucket/manage-webhooks-735643732.html[Bitbucket],
 or Generic webhooks.
 
+[NOTE]
+====
+`oc new-app` and `oc new-build` will create GitHub and Generic webhook triggers
+for you, but you must add any other needed webhook triggers manually (see xref:describe-buildconfig[Displaying Webhook URLs]).
```

Comment:
@gabemontero @oatmealraisin do we not have any docs on how to add triggers to builds (using oc set triggers)?  that is what this should be linking to.  If we don't have them, we should add a subsection covering that tool to this section on webhook triggers.  It should also mention that triggers can be added via the webconsole (pretty sure @jwforres has it on her list to enhance the webconsole to support the 2 new trigger types).